src/test/scala/mesosphere/marathon/api/akkahttp/v2/ResponseMatchers.scala

```diff
@@ -0,0 +1,89 @@
+package mesosphere.marathon
+package api.akkahttp.v2
+
+import mesosphere.UnitTest
+import org.scalatest.matchers.{ HavePropertyMatchResult, HavePropertyMatcher }
+import play.api.libs.json.{ JsNumber, JsObject, JsValue }
+
+trait ResponseMatchers { this: UnitTest =>
+
+  /**
+    * Have property matcher for pods executor resources.
+    *
+    * @param cpus expected CPUs
+    * @param mem expected memory
+    * @param disk expected disk
+    * @return Match result.
+    */
+  def executorResources(cpus: Double, mem: Double, disk: Double) = new HavePropertyMatcher[JsValue, Option[JsValue]] {
+    override def apply(actual: JsValue) = {
+      val maybeActual = (actual \ "executorResources").toOption
+      val expected = JsObject(Seq("cpus" -> JsNumber(cpus), "mem" -> JsNumber(mem), "disk" -> JsNumber(disk)))
+      val matches = maybeActual.contains(expected)
+      HavePropertyMatchResult(matches, "executorResources", Some(expected), maybeActual)
+    }
+  }
+
+  /**
+    * Have property matcher verifying that no networkname has been defined.
+    */
+  val noDefinedNetworkname = new HavePropertyMatcher[JsValue, Option[JsValue]] {
```

Comment:
networkName not networkname ?

Reply:
I think networkname is common enough, no?

Reply:
hmm am I missing something? It is two words, so I would do the networkName (name is a new word...)